### PR TITLE
Our own certificate

### PR DESCRIPTION
**Pull request by pallenSL**
(still secrets in GIT though)

### CODE REVIEW

File: Auth/AppSettings.fs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿module AppSettings
+
+open Serilog
+open System
+open System.Configuration
+
+let getString (key : string) =
+    let value = ConfigurationManager.AppSettings.Get(key)
+    if String.IsNullOrEmpty value then
+        let msg = String.Format("AppSetting '{0}' is null or empty", key)
```
Comment (jebbinSCL) [inline Auth/AppSettings.fs]: F# preference would be for this line to be: 
let msg = `sprintf "AppSetting '%s' is null or empty" key

**Comment by jebbinSCL** [comment]
Can we rename AppSettings.fs and the module to Config for consistency with the other repos?

**Comment by jebbinSCL** [comment]
:shipit:

File: Auth/Startup.fs [hunk 2]
```diff
@@ -69,13 +68,17 @@ type Startup() =
                 IdentityProviders = Action<_,_>(configureIdentityProviders))
 
         let getEmbeddedCertificate() = 
-            use stream = __.GetType().Assembly.GetManifestResourceStream("idsrv3test.pfx")            
+            let file = Config.getString "certificateFile"
+            let pwd = Config.getString "certificatePassword"
+            
+            use stream = __.GetType().Assembly.GetManifestResourceStream(file)
```
Comment (csscottlogic) [inline Auth/Startup.fs]: The __ is a convention meaning "this is never used to refer to the instance". Now that we are referring to the instance, we should use something like ```x``` or ```this```.

**Pull request merged**